components/layout/index.jade

```diff
@@ -6,6 +6,8 @@ html( lang="zh" )
     meta( http-equiv="Content-Type" content="text/html; charset=UTF-8" )
     link( type='text/css', rel='stylesheet', href='//fast.fonts.net/cssapi/f7f47a40-b25b-44ee-9f9c-cfdfc8bb2741.css' )
     link( rel='stylesheet' type='text/css' href='index.css' )
+    script( src='//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js' )
```

Comment:
I would make sure you point to a local copy of jquery instead of a CDN, this is basically the reason why we are hosting this page separately in the first place

![](https://sahaye.files.wordpress.com/2010/03/china_wall_on_google_cartoon.png)